src/Tribe/Views/V2/Hooks.php

```diff
@@ -497,7 +497,7 @@ public function filter_wp_title( $title, $sep = null ) {
 	 */
 	public function pre_get_document_title( $title ) {
 		$bootstrap = $this->container->make( Template_Bootstrap::class );
-		if ( ! $bootstrap->should_load() || $bootstrap->is_single_event() ) {
+		if ( ! $bootstrap->should_load() || $bootstrap->is_single_event() || tribe_context()->is( 'tec_post_type' ) ) {
```

Comment:
**I'd** do the `tec_post_type` test before we even create the bootstrap object (i.e. before line 499)